tsconfig.json

```diff
@@ -18,6 +18,8 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "noUncheckedIndexedAccess": true, // Disallow unchecked access to indexed properties
```

Comment:
The comment in not necessary, most IDE will give you access to the reference of this property.